### DISCUSSION TITLE
Wyze beta fw security improvements

### DISCUSSION THREAD

**Discussion by cheme75** [mrlt8/docker-wyze-bridge#451] [Q&A]
I would not dismiss wyze efforts to break 3rd party access. Saw new v2 cam beta fw 4.9.9.1273 mentions security improvements. Not sure what they could do, but I’m betting they will be trying to lock down their equipment. So anyone test it?

**Comment by mrlt8** [comment]
Installed 4.9.9.1273 on a spare v2 and it's working as expected in the bridge.
